docker-cli.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+jar='kafka-mesos*.jar'
+
+check_jar() {
+    jars=$(find . -maxdepth 1 -name "$jar" | wc -l)
+
+    if [ $jars -eq 0 ]
+    then
+        echo "$jar not found"
+        exit 1
+    elif [ $jars -gt 1 ]
+    then
+        echo "More than one $jar found"
+        exit 1
+    fi
+}
+
+check_environment() {
```

Comment:
This looks great, can you update the README with a section about all of this and how it works please? Thanks!